src/ert/scheduler/local_driver.py

```diff
@@ -26,7 +26,7 @@ async def kill(self, iens: int) -> None:
             self._tasks[iens].cancel()
             await self._tasks[iens]
             del self._tasks[iens]
-        except KeyError:
+        except (KeyError, asyncio.CancelledError):
```

Comment:
We do catch cancellation in `driver._run`, so not sure how it helps here?

Reply:
This helps when we are called from https://github.com/equinor/ert/blob/9b92e9ad9407fd2b81d91e2e3a1570f64fa4faeb/src/ert/scheduler/job.py#L136